bedrock/firefox/templates/firefox/personal.html

```diff
@@ -0,0 +1,67 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+ {% extends "base-resp.html" %}
+
+{% block page_title_prefix %}{{_('Firefox Web Browser')}} - {% endblock %}
+{% block page_title %}{{_('When it&rsquo;s personal, choose Firefox')}}{% endblock %}
+
+{% block page_image %}{{ static('img/firefox/personal/logo.png') }}{% endblock %}
+{% block page_desc %}{{_('Choose Firefox when you want to browse the Web without the Web browsing you.')}}'{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'personal' %}
+{% endblock %}
+
+{% block site_header_nav %}{% endblock %}
+{% block site_header_logo %}{% endblock %}
+
+{% block alt_header %}
+<div class="header-content">
+  <h1>{{_('When it&rsquo;s personal,<span>choose Firefox</span>')}}</h1>
+  {{ download_firefox(simple=true) }}
+
+  <div class="mobile-download">
+    <a href="https://play.google.com/store/apps/details?id=org.mozilla.firefox&referrer=utm_source%3Dmozilla%26utm_medium%3DReferral%26utm_campaign%3Dmozilla-org">
+      <img src="{{ static('img/firefox/personal/google_play_badge.png') }}" width="172" alt="{{_('Download from Google Play')}}" />
+    </a>
+  </div>
+</div>
+{% endblock alt_header %}
+
+{% block extrahead %}{% endblock %}
+
+{% block js_pretrack %}{% endblock %}
+
+{% block content %}
+<main role="main">
+  <div class="mission">
+    <p><span>We believe</span> that your personal<br /> information belongs<br /> only to you.</p>
+    <a href="//mozilla.org/firefox/desktop/trust">See how we put you first</a>
+  </div>
+
+  <div class="mission">
+    <p class="data"><span>We never</span> act in a way that<br /> would compromise<br /> that belief.</p>
+    <a href="//mozilla.org/about/manifesto">Learn about our mission</a>
+  </div>
+
+  <div class="mission">
+    <p class="choice"><span>You choose</span> how your information<br /> is collected and<br /> shared online.</p>
+    <a href="{{ url('privacy.privacy-day') }}">Get tips about your privacy</a>
+  </div>
+</main>
+
+<section class="mobile-download">
+  <p>{{_('Add Firefox to your smartphone or tablet')}}</p>
+  <a href="https://play.google.com/store/apps/details?id=org.mozilla.firefox&referrer=utm_source%3Dmozilla%26utm_medium%3DReferral%26utm_campaign%3Dmozilla-org">
```

Comment:
Again, this link can be replaced by the canonical link to the play store in settings.